website/docs/prysm-usage/fee-recipient.md

```diff
@@ -0,0 +1,76 @@
+---
+id: fee-recipient
+title: Fee Recipient
+sidebar_label: fee recipient
+---
+
+# Fee Recipient
+
+Fee Recipient is the feature that allows users receive gas fees when proposing blocks. These gas rewards are some times know as the inclusion rewards.
+
+:::tip Post Bellatrix Fork ( The Merge )
+This feature will only activate Post Bellatrix Fork. Currently stakers do not receice gas fees from proposing blocks.
+:::
+
+:::warning Known Release Bug
+Setting a default fee recipient through the validator client is currently not working in the upcoming branch a fix was created and merged into develop for the following release. please refer to [#10555](https://github.com/prysmaticlabs/prysm/pull/10555)
+:::
+
+:::warning Be aware of your Fee Recipient Settings!!
+
+Protect and review your Fee Recipient settings.
+
+Fee Recipient settings will default to the burn address ( 0x000....0 ) so it is important to set it to a hex address where you can be paid this. 
+
+Please also make sure the address is yours when it is set, otherwise it will be sent to someone else's address.
+:::
+
+## New Flags
+
+
+## Validator Client
+- `suggested-fee-recipient` to set default eth address for ALL validators.
+( i.e. --suggested-fee-recipient=0x046Fb65722E6b0000012BFEBf6177F1D2e9758D9)
+- `fee-recipient-config-file` to set file location for fee recipient json. 
+( i.e. --fee-recipient-config-file=./fee_recipient_config.json)
+- `fee-recipient-config-url` to set URL for fee recipient json.
+( i.e. --fee-recipient-config-url=http://example.com/api/getFeeRecipient)
+
+suggested fee recipient flag will override the configuration file.
+
+you can not use both the file and url flags at the same time.
+
+the validator client will send the fee recipient information at startup for all public keys that are active
+as well as when the public key becomes active. It does this by calling the beacon api `prepareBeaconProposer`
+
+### Fee Recipient Config File based on Teku Implementation
```

Comment:
When does a reader need to care about this section?

Reply:
if they use the fee recipient config file or url this just shows the format for that file.